contracts/ConvergentCurvePool.sol

```diff
@@ -696,6 +637,46 @@ contract ConvergentCurvePool is IMinimalSwapInfoPool, BalancerPoolToken {
         revert("Token request doesn't match stored");
     }
 
+    /// @notice Mutates the input current balances array to remove fees paid to governance and balancer
+    ///         Also resets the LP storage and realizes governance fees, returns the fees which are paid
+    ///         to balancer
+    /// @param currentBalances The overall balances
+    /// @param balancerFee The percent of LP fees that balancer takes as a fee
+    /// @return dueProtocolFees The fees which will be paid to balancer on this join or exit
+    /// @dev WARNING - Solidity will implicitly cast a 'uint256[] calldata' to 'uint256[] memory' on function calls, but
+    ///                since calldata is immutable mutations made in this call are discarded in future references to the
+    ///                variable in other functions. 'currentBalances' must STRICTLY be of type uint256[] memory for this
+    ///                function to work.
+    function _feeAccounting(
+        uint256[] memory currentBalances,
+        uint256 balancerFee
+    ) internal returns (uint256[] memory dueProtocolFees) {
+        // Load the total fees
+        uint256 localFeeUnderlying = uint256(feesUnderlying);
+        uint256 localFeeBond = uint256(feesBond);
+        dueProtocolFees = new uint256[](2);
+        // Calculate the balancer fee [this also implicitly returns this data]
+        dueProtocolFees[bondIndex] = localFeeUnderlying.mulDown(balancerFee);
+        dueProtocolFees[baseIndex] = localFeeBond.mulDown(balancerFee);
```

Comment:
The `baseIndex` and `bondIndex` are reversed here:
```suggestion
        dueProtocolFees[baseIndex] = localFeeUnderlying.mulDown(balancerFee);
        dueProtocolFees[bondIndex] = localFeeBond.mulDown(balancerFee);
```

Reply:
Very good catch! Thanks!